hack/chains/config.sh

```diff
@@ -78,6 +78,33 @@ case "$1" in
 
     ;;
 
+  rekor-local )
+    REKOR_SERVER=$(kubectl get ingress -n rekor-server -o yaml|yq e '.items[.spec].spec.rules[.host].host')
+    $0 "transparency.url: \"$REKOR_SERVER\"" $2
+
+    ;;
+
+  rekor-default )
+    # if setting rekor-default we should ensure we don't have a transparency.url value
+    $0 remove-key 'transparency.url' $2
+
+    ;;
+
+  remove-key )
+    # Remove the given key from the data section
```

Comment:
If you're worried about dry run support how about a comment:

    # Beware there's no dry-run mode for removing keys

Reply:
Or..

        if [[ $3 == "--dry-run" ]]; then
           echo "Removing $2"
        else
           ...